client/src/login/Login.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import axios from 'axios';
+
+class Login extends React.Component {
+  state = {
+    username: '',
+    password: ''
+  }
+
+  onInputChange = event => {
+    const { name, value } = event.target;
```

Comment:
There are a few alternative ways you could use destructuring here to the same effect:
- taking target off of the event parameter (so as to slightly reduce repetition)
```javascript
onInputChange = ({ target }) => {
  const { name, value }  = target;
  this.setState({
    [name]: value
  })
};
```
- or, take off name and value since you don't need target (note that you _won't_ have access to the target object here, but only the innermost properties you destructure off, in this case name and value):
```javascript
onInputChange = ({ target: { name, value } }) => {
  this.setState({
    [name]: value
  })
};
```